composer.json

```diff
@@ -31,7 +31,7 @@
     ],
     "homepage": "https://github.com/spatie/activitylog",
     "require": {
-        "php": "^8.0",
+        "php": "^8.0|^8.1",
```

Comment:
`^8.0` includes PHP `8.1`
https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

Reply:
Thanks got it well